sdcm/nemesis.py

```diff
@@ -1397,22 +1401,27 @@ class NonDisruptiveMonkey(Nemesis):
         #  - NoCorruptRepairMonkey
         #  - MgmtRepair
         #  - AbortRepairMonkey
+
+    def __init__(self, tester_obj, termination_event):
```

Comment:
I think better to get `*args, **kwargs` since if we change signature of Nemesis constructor we won't need to remember to change here

Reply:
fixed